### PR TITLE
Feature/include datadog log parser

### PR DESCRIPTION
**Pull request by anderruiz**
## What does it do?

It includes a new parser to be able to get information for the new Datadog vulnerability detection

### CODE REVIEW

File: plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java
Comment (darkspirit510) [inline plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java:15]: Could you change this name, it's not my test 🙈

Comment (anderruiz) [reply plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java:15]: Ok, copied from my original HdivReaderTest :)

**Comment by davewichers** [comment]
@anderruiz - can you point me to the web page for this tool?

**Comment by anderruiz** [comment]
Datadog platform is available here:

https://www.datadoghq.com/

It is a commercial solution, and vulnerability detection is not released yet to the public. The engine will be based in Hdiv one which parser I included in the Benchmark project 5 years ago.

File: plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java
Comment (darkspirit510) [inline plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java:42]: You mentioned the scanner is based on Hdiv, but it's an own Reader. So I guess the test name should match the reader (just cosmetics 😅)

**Comment by darkspirit510** [comment]
Looks good to me. I can create the scorecard with the testfile without errors. Could you send me a more complex testfile to github@darkspirit510.de?

**Comment by anderruiz** [comment]
Sent a more complex log file!

File: plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java
Comment (anderruiz) [reply plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/DatadogReaderTest.java]: Done!

**Comment by darkspirit510** [comment]
Thanks, runs without any issues.

@davewichers guess this one can be merged

**Review by darkspirit510 — APPROVED** [review]

**Comment by anderruiz** [comment]
@davewichers Any other comment or thing you want to clarify?

**Comment by davewichers** [comment]
Hey - sorry for the delay. I was on holiday for 2 weeks then caught Covid :-(. Anyway, is this product publicly available yet. If not, I'd prefer to hold off until it is public to merge it in. Then I'll update the OWASP site to indicate this tool is now supported.

**Comment by davewichers** [comment]
Any news on when the product at: https://www.datadoghq.com/ is or will be released? And they have a lot of products, so what is the name of this product, or the product that includes this type of scanning?

**Comment by davewichers** [comment]
@anderruiz - Can you answer my previous question?

**Comment by anderruiz** [comment]
Sorry for the delay. The product page is this one:

https://www.datadoghq.com/product/application-vulnerability-management/

Do you think we could merge the PR?

**Pull request merged**
